presto-main/src/main/java/com/facebook/presto/operator/PartitionedOutputOperator.java

```diff
@@ -445,65 +441,65 @@ public void flush(boolean force)
         }
     }
 
-    public static class PartitionedOutputInfo
-            implements Mergeable<PartitionedOutputInfo>, OperatorInfo
-    {
-        private final long rowsAdded;
-        private final long pagesAdded;
-        private final long outputBufferPeakMemoryUsage;
-
-        @JsonCreator
-        public PartitionedOutputInfo(
-                @JsonProperty("rowsAdded") long rowsAdded,
-                @JsonProperty("pagesAdded") long pagesAdded,
-                @JsonProperty("outputBufferPeakMemoryUsage") long outputBufferPeakMemoryUsage)
-        {
-            this.rowsAdded = rowsAdded;
-            this.pagesAdded = pagesAdded;
-            this.outputBufferPeakMemoryUsage = outputBufferPeakMemoryUsage;
-        }
-
-        @JsonProperty
-        public long getRowsAdded()
-        {
-            return rowsAdded;
-        }
-
-        @JsonProperty
-        public long getPagesAdded()
-        {
-            return pagesAdded;
-        }
-
-        @JsonProperty
-        public long getOutputBufferPeakMemoryUsage()
-        {
-            return outputBufferPeakMemoryUsage;
-        }
-
-        @Override
-        public PartitionedOutputInfo mergeWith(PartitionedOutputInfo other)
-        {
-            return new PartitionedOutputInfo(
-                    rowsAdded + other.rowsAdded,
-                    pagesAdded + other.pagesAdded,
-                    Math.max(outputBufferPeakMemoryUsage, other.outputBufferPeakMemoryUsage));
-        }
-
-        @Override
-        public boolean isFinal()
-        {
-            return true;
-        }
-
-        @Override
-        public String toString()
-        {
-            return toStringHelper(this)
-                    .add("rowsAdded", rowsAdded)
-                    .add("pagesAdded", pagesAdded)
-                    .add("outputBufferPeakMemoryUsage", outputBufferPeakMemoryUsage)
-                    .toString();
-        }
-    }
+//    public static class PartitionedOutputInfo
```

Comment:
- delete commented out code
- extract this change into a separate commit: `Extract PartitionedOutputInfo as a top-level class`